glide.lock

```diff
@@ -256,7 +260,7 @@ imports:
 - name: github.com/spf13/jwalterweatherman
   version: 33c24e77fb80341fe7130ee7c594256ff08ccc46
 - name: github.com/spf13/pflag
-  version: a9a634f3de0a7529baded7ad6b0c7467d5c6eca7
+  version: 5644820622454e71517561946e3d94b9f9db6842
```

Comment:
this change breaks the CLI build. a9a634f is newer than 5644820, and has the fields that are missing in the older version (vendor/github.com/spf13/cobra/bash_completions.go:370: flag.NoOptDefVal undefined (type *pflag.Flag has no field or method NoOptDefVal))